clippy_lints/src/fallible_impl_from.rs

```diff
@@ -18,13 +18,24 @@ declare_clippy_lint! {
     /// **Known problems:** None.
     ///
     /// **Example:**
-    /// ```rust
+    /// ```rust,ignore
```

Comment:
Why is this ignore?

Reply:
I'm getting an error:
```error[E0119]: conflicting implementations of trait `std::convert::TryFrom<std::string::String>` for type `main::Foo`:```

Seems to be related to: https://github.com/rust-lang/rust/issues/50133

I will try to find a workaround

Reply:
You could split up the code blocks and redefine  `Foo` with a leading `#` in the second block.